index.html

```diff
@@ -20,71 +20,223 @@
+              <li class="text-400 roboto align-left li-color">Codekit</li>
+              <li class="text-400 roboto align-left li-color">GitHub</li>
+              <li class="text-400 roboto align-left li-color">Codepen</li>
+              <li class="text-400 roboto align-left li-color">GitLab</li>
+              <li class="text-400 roboto align-left li-color">Terminal</li>
+            </ul>
+          </div>
+        </div>
+        <hr class="horizontal" />
+        <div class="socials">
+          <a href="https://github.com/itsmraga-hub"
+            ><img src="assets/icons/github.png" alt="github logo"
+          /></a>
+          <a href="https://github.com/itsmraga-hub"
+            ><img src="assets/icons/linkedin.png" alt="linkedin logo" />
+          </a>
+          <a href="">
+            <img src="assets/icons/fingers.png" alt="two fingers held up" />
+          </a>
+          <a href="https://twitter.com/RagaMacharia"
+            ><img src="assets/icons/twitter.png" alt="teitter logo"
+          /></a>
+          <a href="">
+            <img src="assets/icons/M.png" alt="letter M" />
+          </a>
+        </div>
+        <div class="nav-arrow">
+          <a href="#work"
+            ><img class="arrow bow" src="assets/projects/arrow.png" alt="" />
+          </a>
+          <a href="#work" class="works"
+            ><p class="work roboto align-center">Work</p></a
+          >
+        </div>
+      </section>
+      <section class="grid-container">
+        <div class="grid-item">
+          <h3 class="head-color roboto pr-title" id="work">MY RECENT WORK</h3>
+        </div>
+        <hr class="horizontal" />
+        <div class="grid-item first">
+          <div class="grid-row-item">
+            <img
+              src="assets/projects/Img-Placeholder.png"
+              alt="Project Image"
+              class="project-image"
+            />
+          </div>
+          <div class="grid-row-item">
+            <h4 class="project-title roboto white">Multi-Post Stories</h4>
+          </div>
+          <div class="white-background"></div>
+          <div class="grid-row-item">
+            <p class="inter text-400 align-left">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required, has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+          </div>
+          <div class="grid-row-item">
+            <ul class="lists">
+              <li class="inter white li-border">css</li>
+              <li class="inter white li-border">html</li>
+              <li class="inter white li-border">bootstrap</li>
+              <li class="inter white">Ruby</li>
+            </ul>
+          </div>
+          <div class="grid-row-item">
+            <button type="button" class="btn btn-primary roboto white">
+              See Project
+            </button>
```

Comment:
- [ ] Great Job implementing the buttons, but you seem to have missed the button interactions specified in the Figma template. You can check it out in the Figma template, please do implement them. I've added an image below as a reference

![image](https://user-images.githubusercontent.com/90556221/176886962-c776ea35-a272-4042-9ef6-3018a183d078.png)